ui/app/templates/vault/cluster/clients/counts/overview.hbs

```diff
@@ -5,7 +5,7 @@
 
 <Clients::Page::Overview
   @activity={{this.model.activity}}
-  @isSecretsSyncActivated={{this.model.isSecretsSyncActivated}}
+  @showSecretsSync={{this.model.showSecretsSync}}
```

Comment:
I _think_ this arg can be deleted because the child components get their boolean from the flag service 🧹